src/styles/styles.scss

```diff
@@ -3,6 +3,46 @@
 
 $sideborder-margin: 6px;
 
+.selMode.done {
```

Comment:
I don't think these are necessary :p

Reply:
I start from the master, but I don't know why it keeps adding this. X(